op-node/eth/types.go

```diff
@@ -314,3 +314,12 @@ func (f FetchedReceipts) Result() (types.Receipts, error) {
 }
 
 var _ ReceiptsFetcher = (FetchedReceipts)(nil)
+
+// SystemConfig represents the rollup system configuration that carries over in every L2 block,
+// and may be changed through L1 system config events.
```

Comment:
Its a bit error prone to have the initial config in the rollup config and passed to the contract constructor, i don't think there is anything preventing them from being different

Reply:
It seems complex to have to read the initial config from the contract

Reply:
To load an initial configuration from L1 we need to add more code, and discuss how we can read it securely, even when the systemconfig has already changed after genesis. Let's do that in a separate PR, and maybe write a short design doc together how we load the initial config? Opened ENG-2963 to track this.